app/services/page_updater.rb

```diff
@@ -78,11 +78,30 @@ def refresh?
 
   private
 
+  def important_changes_made
+    page_tags_before = @page.pages_tags.map(&:tag_id)
+
+    yield if block_given?
+
+    page_tags_after = @page.pages_tags.map(&:tag_id)
+
+    @page.changed_attributes.keys.any? do |attr|
+      ['language_id', 'title'].include?(attr)
```

Comment:
why not include `'slug'` in this list? We haven't build the tools to edit them yet, but if we do it would be good if it didn't catch this method unawares.